index.html

```diff
@@ -66,207 +71,65 @@ <h2>Requirements</h2>
     <section>
       <h3>Overall</h3>
 
-      <p>[[!ttml-imsc1.1]] SHALL retain the scope of [[!ttml-imsc1.0.1]] while adding the minimal set of features necessary to
+      <p>[[!ttml-imsc1.2]] SHALL retain the scope of [[!ttml-imsc1.1]] while adding the minimal set of features necessary to
       support current practices for worldwide subtitling and captioning delivery.</p>
     </section>
 
     <section>
-      <h3>IMSC1.0.1 Compatibility</h3>
-
-      <p>A valid [[!ttml-imsc1.0.1]] Text Profile document SHALL be a valid [[!ttml-imsc1.1]] Text Profile document, and a valid
-      [[!ttml-imsc1.0.1]] Image Profile document SHALL be a valid [[!ttml-imsc1.1]] Image Profile document.</p>
-
-      <p>A [[!ttml-imsc1.1]] presentation processor SHALL present a valid [[!ttml-imsc1.0.1]] document as it would have been
-      presented by an [[!ttml-imsc1.0.1]] presentation processor.</p>
+      <h3>Open issues</h3>
 
-      <p>The <code>#fillLineGap</code> and <code>#activeArea</code> specified in [[!ttml-imsc1.0.1]] SHALL be permitted in
-      [[!ttml-imsc1.1]].</p>
+      <p>[[!ttml-imsc1.2]] SHALL address <a href="https://github.com/w3c/imsc/issues">outstanding open issues</a> documented in the
+      IMSC repository.</p>
     </section>
 
     <section>
-      <h3>TTML2 Subset</h3>
-
-      <p>A valid [[!ttml-imsc1.1]] Document SHALL be a valid [[!TTML2]] document.</p>
+      <h3>IMSC1.1 Compatibility</h3>
 
-      <p>Any [[!ttml-imsc1.0.1]] extension that (i) is not one of the following and (ii) has an equivalent feature in [[!TTML2]]
-      SHALL be <a>deprecated</a> in [[!ttml-imsc1.1]] and the corresponding [[!TTML2]] feature permitted:</p>
+      <p>A valid [[!ttml-imsc1.1]] Text Profile document SHALL be a valid [[!ttml-imsc1.2]] Text Profile document, and a valid
+      [[!ttml-imsc1.1]] Image Profile document SHALL be a valid [[!ttml-imsc1.2]] Image Profile document.</p>
 
-      <ul>
-        <li><code>#activeArea</code></li>
-
-        <li><code>#fillLineGap</code></li>
-
-        <li><code>#forcedDisplay</code></li>
+      <p>A [[!ttml-imsc1.2]] presentation processor SHALL present a valid [[!ttml-imsc1.1]] document as it would have been
+      presented by an [[!ttml-imsc1.1]] presentation processor.</p>
+    </section>
 
-        <li><code>#linePadding</code></li>
+    <section>
+      <h3>TTML2 Subset</h3>
 
-        <li><code>#multiRowAlign</code></li>
-      </ul>
+      <p>[[!ttml-imsc1.2]] SHALL use features defined in [[!TTML2]] to satisfy each requirement specified herein, unless
+      infeasible.</p>
 
-      <p>When an [[!ttml-imsc1.0.1]] extension is deprecated and an equivalent [[TTML2]] feature is permitted, a document instance
+      <p>When an [[!ttml-imsc1.1]] extension is deprecated and an equivalent [[TTML2]] feature is permitted, a document instance
       SHALL NOT use both the deprecated extension and the equivalent [[TTML2]] feature.</p>
```

Comment:
Question: this reminded me that we have deprecated parts, and one day presumably we will remove them. But the language above about retaining compatibility suggests that in 1.2 we can not remove any deprecated stuff. Is that your intent @palemieux ?

Reply:
At this point, and given the short elapsed time since IMSC 1.1, I am not convinced it is worth the time and risk to remove deprecated features.